hack/generator/pkg/codegen/pipeline_check_for_anytype.go

```diff
@@ -16,22 +16,33 @@ import (
 	"github.com/Azure/k8s-infra/hack/generator/pkg/astmodel"
 )
 
-// checkForAnyType returns a stage that will check for any defs
-// containing AnyTypes. It accepts a set of packages that we expect to
-// contain types with AnyTypes. Those packages will be quietly
-// filtered out of the output of the stage, but if there are more
-// AnyTypes in other packages they'll be reported as an error. The
-// stage will also return an error if there are packages that we
-// expect to have AnyTypes but turn out not to.
-func checkForAnyType(packages []string) PipelineStage {
+// filterOutDefinitionsUsingAnyType returns a stage that will check for any definitions
+// containing AnyTypes. It accepts a set of packages that we expect to contain types
+// with AnyTypes. Those packages will be quietly filtered out of the output of the
+// stage, but if there are more AnyTypes in other packages they'll be reported as an
+// error. The stage will also return an error if there are packages that we expect
+// to have AnyTypes but turn out not to, ensuring that we clean up our configuration
+// as the schemas are fixed and our handling improves.
+func filterOutDefinitionsUsingAnyType(packages []string) PipelineStage {
+	return checkForAnyTypeCore("Filter out rogue definitions using AnyTypes", packages)
+}
+
+// ensureDefinitionsDoNotUseAnyTypes returns a stage that will check for any
+// definitions containing AnyTypes. The stage will return errors for each type
+// found that uses an AnyType.
+func ensureDefinitionsDoNotUseAnyTypes() PipelineStage {
+	return checkForAnyTypeCore("Catch rogue definitions using AnyTypes", []string{})
+}
+
+func checkForAnyTypeCore(description string, packages []string) PipelineStage {
```

Comment:
Maybe just `checkForAnyType`? Then it's still a verb phrase.

Reply:
I renamed it early on when intending to retain `checkForAnyType()` as the name of the method that created the stage; now that I'm using `filter`/`ensure` for those methods, changing this back makes sense.